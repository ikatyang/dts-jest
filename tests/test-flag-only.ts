declare const test_flag_only: (v: any) => {result: any};

// @dts-jest
test_flag_only(1);

// @dts-jest:only
test_flag_only(2);

// @dts-jest
test_flag_only(3);
