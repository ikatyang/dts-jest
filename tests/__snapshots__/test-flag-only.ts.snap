// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test_flag_only(2) 1`] = `"{ result: any; }"`;
